### DISCUSSION TITLE
Looking for Professional Help? Check the official StackStorm Partners

### DISCUSSION THREAD

**Discussion by st2stanley** [StackStorm/st2#5519] [General]
This is an open-source community and you'll find here contributors answering your questions during their free time.

If you're looking for a more dedicated professional help and need support with SLAs, custom paid solutions, training, or more, we recommend reaching to one of the official StackStorm Partners here:
### https://stackstorm.com/partners/

> **NOTE**
Partners have substantial experience with the StackStorm and they also give back to the community by helping maintaining the project via monetary contributions or submitting the features, fixes, and improvements back to the platform.

**Comment by Adip-BTS** [comment]
Hello StackStorm Team,
I was trying to use azure pack of stackstorm to integrate it with Azure Monitor service. But, Stackstorm consists of only 22 actions and none of them are useful to integrate with Azure Monitor service. Can you please provide guidance on this? It will be very helpful. Iam using open source version of Stackstorm and configured it using docker.

Thank You.

**Reply by arm4b** [reply]
I guess you mean https://github.com/StackStorm-Exchange/stackstorm-azure Azure integration pack from the https://exchange.stackstorm.org/.

If there is missing something, you can develop and contribute the functionality you need for Azure pack. Alternatively, hire a partner (see https://github.com/StackStorm/st2/discussions/5519#discussion-3768012 message above) who will do it for you.
